shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 
 export const PageMetaHeaderDocket = ({
+  addedDocketNumbers = [],
   docketNumber,
   // do not set this to true unless the content of the PDF also is rendering the century schoolbook font; there is a puppeteer documented here https://github.com/puppeteer/puppeteer/issues/422#issuecomment-759424240
```

Comment:
@Mwindo Make this comment grammatical and resolve after (waiting to do this in case there are other changes to make)